src/Services/Kitchen/Kitchen/Program.cs

```diff
@@ -32,8 +34,15 @@ private static void Main(string[] args)
 
         var builder = new ServiceCollection()
             .AddScoped<IKitchenService, KitchenService>()
+            .AddDbContext<KitchenDbContext>(options =>
+            {
+                options.UseSqlite("Data Source=sql.db");
+            })
+            .AddSingleton<IKitchenRepository, KitchenRepository>()

```

Comment:
Make this use appsettings:
Prod => prodconnection (in k8s)
dev => devconnection (sqlite)

Reply:
Will do in new branch. Ok for now.